src/SFA.DAS.ContentApi/Application/Queries/GetContentQuery/GetContentQuery.cs

```diff
@@ -0,0 +1,11 @@
+﻿using MediatR;
+
+namespace SFA.DAS.ContentApi.Application.Queries.GetContentQuery
+{
+    public class GetContentQuery : IRequest<string>
+    {
+        public string Type { get; set; }
+
+        public string ClientId { get; set; }
```

Comment:
Need to think about a naming format/strategy.  Maybe add a comment here about what format to follow.

Reply:
Agree. Could've used enums but would've meant coding would be required if new clients / content types are added. The ReadMe can be used to show valid values.

Reply:
A string is better than an enum due to less coupling.  I raised it because the first ones usually dictate the following and are not likely to change once implemented.  So  are we OK with acc and  acc_legacy or do we need a more formal format?

Reply:
They're just temporary until the DB project is added. Will all be in data then so can be decided later or by individual teams.

Reply:
Ok ,  thought they were going to be used by Vas in her story?

Reply:
might be good to go down the route of eas_{sub-app}, so eas_acc, eas_fin, eas_comt then the provider stuff can follow suit with pas_{sub-app}